doc/release-notes/1.0/1.0.10.rst

```diff
@@ -0,0 +1,37 @@
+======================
+ Percona Agent 1.0.10
+======================
+
+Percona is glad to announce the release of Percona Agent 1.0.10 on December 30th 2014. Downloads are available from our download site `here <http://www.percona.com/downloads/percona-agent/1.0.10>`_. Percona Agent can be `installed <http://cloud-docs.percona.com/Install.html>`_ or `upgraded <http://cloud-docs.percona.com/Install.html#updating-the-agent>`_ as described in the `manual <http://cloud-docs.percona.com/index.html>`_. 
+
+New Features
```

Comment:
Can you please provide more info (or ticket-IDs) on why/how this was implemented?